elements/pl-checkbox/pl-checkbox.mustache

```diff
@@ -85,7 +85,7 @@
     
 {{#answers}}
     <li {{#inline}}class="list-inline-item"{{/inline}}>
-    ({{key}}) {{{html}}}
+    {{^hide_letter_keys}}({{key}}){{/hide_letter_keys}} {{{html}}}
 
     {{#display_score_badge}}
         {{#correct}}
```

Comment:
Missing the markup to hide the keys in the answer panel on lines 122-132 (it didn't let me select the lines here for comment)

Reply:
fixed in 60cb8d3ca1c51570d4e425e70d5afa2ad12ec5f0